### ISSUE TITLE
Compatibility issue with phpBB 3.3.0

### ISSUE THREAD

**Issue by luilegeant** [marttiphpbb/phpbb-ext-topicprefixtags#1]
## Description
Installing and enabling this extension on a phpBB 3.3.0 board doesn't work. ;-)
The topic listing fails and returns a blank page.

## How to reproduce:
1. Setup a phpBB board version 3.3.0.
2. install and enable the following extensions:
  - [phpextcalendar](https://github.com/marttiphpbb/phpbb-ext-phpextcalendar/releases/download/0.1.0/phpextcalendar_0_1_0.zip)
  - [calendarmono](https://github.com/marttiphpbb/phpbb-ext-calendarmono.git)
  - [jqueryuidatepicker](https://github.com/marttiphpbb/phpbb-ext-jqueryuidatepicker.git)
  - [calendarmonoinput](https://github.com/marttiphpbb/phpbb-ext-calendarmonoinput.git)
  - [calendartag](https://github.com/marttiphpbb/phpbb-ext-calendartag.git)
  - [menuitems](https://github.com/marttiphpbb/phpbb-ext-menuitems.git)
3. install this extension, configure it in the ACP
4. navigate to your board home page, visit a subforum (where your expect a list of topics) and the page stays blank.

## Logs:
```
2020-01-09T10:47:03.172721010Z 2020/01/09 10:47:03 [error] 6#6: *175 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined method phpbb\event\data::getName() in /var/www/forum/ext/marttiphpbb/topicprefixtags/event/listener.php:69
2020-01-09T10:47:03.172769991Z Stack trace:
2020-01-09T10:47:03.172787486Z #0 /var/www/forum/vendor/symfony/event-dispatcher/EventDispatcher.php(214): marttiphpbb\topicprefixtags\event\listener->core_viewforum_modify_topicrow(Object(phpbb\event\data), 'core.viewforum_...', Object(phpbb\event\dispatcher))
2020-01-09T10:47:03.172802703Z #1 /var/www/forum/vendor/symfony/event-dispatcher/EventDispatcher.php(44): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'core.viewforum_...', Object(phpbb\event\data))
2020-01-09T10:47:03.172816600Z #2 /var/www/forum/phpbb/event/dispatcher.php(62): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('core.viewforum_...', Object(phpbb\event\data))
2020-01-09T10:47:03.172830371Z #3 /var/www/forum/phpbb/event/dispatcher.php(46): phpbb\event\dispatcher->dispatch('core.viewforum_...', Object(phpbb\event\data))
2020-01-09T10:47:03.172843487Z #4 /var/www/forum/viewforum.php(1006): phpbb\event\dispatcher->trigger_event('core.viewforum_...', Array)
2020-01-09T10:47:03.172856429Z #5 {main}
2020-01-09T10:47:03.172868241Z   throw" while reading response header from upstream, client: 172.18.0.1, server: , request: "GET /forum/viewforum.php?f=2 HTTP/1.1", upstream: "fastcgi://10.0.2.7:9000", host: "localhost", referrer: "http://localhost/forum/index.php?sid=6b03ee18600bcedac43c799b41ed3cd9"
2020-01-09T10:47:03.173605303Z 172.18.0.1 - - [09/Jan/2020:10:47:03 +0000] "GET /forum/viewforum.php?f=2 HTTP/1.1" 500 5 "http://localhost/forum/index.php?sid=6b03ee18600bcedac43c799b41ed3cd9" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:71.0) Gecko/20100101 Firefox/71.0" "-"
```

**Comment by marttiphpbb** [comment]
Thanks for reporting. It should be fixed now in version 0.4.0

**Comment by luilegeant** [comment]
I confirm that in my context it is now fixed. Thanks for the fast response.
